### PR TITLE
feat(back-end): Created minimal server

### PR DESCRIPTION
**Pull request by ValantineSuh**
I created a custom.env file in the backend folder, in which all environmental variables specific to the project will be put.
I created a src folder in the backend, in which a file (index.js) holding the codes for the minimal server is found.

Please @stephane-segning kindly review this PR

This ticket is for;
- #9

### CODE REVIEW

File: expense-backend/src/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Importing 'express' module and 'dotenv' module
+const express = require('express');
+const dotenv = require('dotenv');
+
+dotenv.config({ path: 'custom.env' });
```
Comment (stephane-segning) [inline expense-backend/src/index.js:5]: Don't use a custom.env, use the simple one.

File: expense-backend/custom.env
Comment (stephane-segning) [inline expense-backend/custom.env:1]: Rename this file to `.env`

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work. Please just adjust these points

File: expense-backend/src/index.js (hunk 1 above)
Comment (ValantineSuh) [reply expense-backend/src/index.js:5]: Ok Sir, i'll change it

Comment (ValantineSuh) [reply expense-backend/src/index.js:5]: Please Sir @stephane-segning is it proper to push  `.env` file ?

Comment (stephane-segning) [reply expense-backend/src/index.js:5]: No. It'll stay local. This should be documented for every developer to use at create one such file before he starts working properly

Comment (ValantineSuh) [reply expense-backend/src/index.js]: Ok Sir, noted.

**Review by bryandino673 — APPROVED** [review]

**Pull request merged**